src/OpenTelemetry.Exporter.Prometheus/Implementation/PrometheusSerializerExt.cs

```diff
@@ -133,6 +133,9 @@ public static int WriteMetric(byte[] buffer, int cursor, Metric metric)
                     }
 
                     // Histogram sum
+                    var count = metricPoint.GetHistogramCount();
```

Comment:
Do we need to declare these variables here, or just use them inline?

Reply:
Fixed this in #2664